doc/classes/TextServer.xml

```diff
@@ -2199,6 +2352,33 @@
 		<constant name="SUBPIXEL_POSITIONING_ONE_QUARTER_MAX_SIZE" value="16" enum="SubpixelPositioning">
 			Maximum font size which will use "one quarter of the pixel" subpixel positioning in [constant SUBPIXEL_POSITIONING_AUTO] mode.
 		</constant>
+		<constant name="DRAW_CALL_NULL" value="0" enum="DrawCallType">
+			Draw call that do nothing.
```

Comment:
[Gimme a cheese with nothing](https://youtu.be/BG7273yDpdA?si=MOQ9mCTt8FuTp36z&t=9)
```suggestion
			Draw call that does nothing.
```
Also perhaps stating what the actual purpose of this is. I am to assume it's an initial default value, similar to `KEY_NONE`.

Reply:
>  I am to assume it's an initial default value, similar to KEY_NONE.

It is.